source/funkin/menus/TitleState.hx

```diff
@@ -119,6 +119,13 @@ class TitleState extends MusicBeatState
 	{
 		if (FlxG.keys.justPressed.F)  FlxG.fullscreen = !FlxG.fullscreen;
 
+		#if desktop
+		if (FlxG.keys.justPressed.ESCAPE)
+		{
+			lime.system.System.exit(0);
+		}
+		#end
+
```

Comment:
Why, also remove this

Reply:
Well, I was thinking to implement some  functions from the Base Game, but now I realized that it can be implemented as a mod. My bad

Reply:
Ok, it's solved :3